### ISSUE TITLE
Doesn't work in Chrome any more :(

### ISSUE THREAD

**Issue by daniellmb** [adriancooney/console.image#13]
:cry:

**Comment by adriancooney** [comment]
:scream: What version/OS you running? [The demo](http://adriancooney.ie/console.image/) is still working for me on Version 44.0.2403.130 (64-bit) OSX Yosemite!

**Comment by daniellmb** [comment]
Chrome 45.0.2454.26 beta (64-bit) OSX 10.9.5

It looks like the code runs without errors but Chrome isn't rendering the BG image. It'll render a BG color, but not an image (tried both a url and base 64 encoded string).

![screen shot 2015-08-10 at 10 06 14 am](https://cloud.githubusercontent.com/assets/108018/9177715/af51253a-3f47-11e5-9337-1188013e9996.png)

**Comment by k4rm3l0** [comment]
Doesn't work in Chrome - Version 45.0.2454.85 (64-bit) / OSX 10.10.5

**Comment by mudcube** [comment]
Unfortunately it appears the fun is over :(

https://code.google.com/p/chromium/issues/detail?id=495643

I believe the only way to get around would be to draw pixel-by-pixel... bonus of that would be that it would work in Firefox and Opera too ;) I think it would be intense on large images though.

**Comment by mudcube** [comment]
Just tried to do the pixel-by-pixel method; it's super slow, and could not get pixels to align properly.

Better option might be to write a DevTools extension; 

https://developer.chrome.com/extensions/devtools

**Comment by jondcallahan** [comment]
:sob:

**Comment by daniellmb** [comment]
Yeah this line kills it: https://src.chromium.org/viewvc/blink/trunk/Source/devtools/front_end/console/ConsoleViewMessage.js?pathrev=197345#l797

There's no way to specify an image without using `url(...)` is there? Even a base64 data uri uses that. :(

**Comment by kumavis** [comment]
the saddest day

**Comment by adriancooney** [comment]
Think we might be back in business.

https://docs.google.com/document/d/1FTascZXT9cxfetuPRT2eXPQKXui4nWFivUnS_335T3U/preview

**Comment by daniellmb** [comment]
``` javascript
console.log([
"     .-.    Since it's not enabled by default,",
"    /_ _\\   I think I shall just use ASCII Art.",
"    |o^o|  /",
"    \\ _ /",
"   .-'-'-.",
" /`)  .  (`\\",
"/ /|.-'-.|\\ \\",
"\\ \\| (_) |/ /  .-\"\"-.",
" \\_\\'-.-'/_/  /[] _ _\\",
" /_/ \\_/ \\_\\ _|_o_LII|_",
"   |'._.'|  / | ==== | \\",
"   |  |  |  |_| ==== |_|",
"    \\_|_/    ||\" ||  ||",
"    |-|-|    ||LI  o ||",
"    |_|_|    ||'----'||",
"   /_/ \\_\\  /__|    |__\\"
].join("\n"));
```

**Comment by adriancooney** [comment]
@daniellmb don't lose hope just yet. I think we're onto something because these custom formatters are awesome. Everybody will probably in the near future have them turned on and we'll be there to pounce.

<img width="801" alt="screenshot 2016-01-30 15 16 21" src="https://cloud.githubusercontent.com/assets/621110/12696517/c4aa3bd8-c764-11e5-8e9f-98002d76adcd.png">

**Comment by mudcube** [comment]
WhatWG is working to make some more standardizations surrounding window.console (good place to weigh in on console.image and why it's important):  https://github.com/whatwg/console

**Comment by leocabral** [comment]
"As far as there are multiple different ways to detect opened devtools, (and we don't aim to fight with them ATM), closing this as WontFix."

from: https://bugs.chromium.org/p/chromium/issues/detail?id=495643

hoorayy!! 🎉🎉🎉

**Comment by bharatramnani94** [comment]
The demo works with Version 63.0.3239.
Maybe it's time to close this issue.

**Comment by firecrauter** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=1350078
https://bugs.chromium.org/p/chromium/issues/detail?id=1223475
https://github.com/GoogleChrome/developer.chrome.com/issues/3386

**Comment by leobenkel** [comment]
Is there an alternative way to display an image in the console nowadays?

**Comment by mpaw** [comment]
@leobenkel This method still works
